languages/swift/exercises/concept/string-components/.docs/introduction.md

```diff
@@ -0,0 +1,31 @@
+We've seen how to build up strings in previous exercises. Here we will see some of the ways to break strings apart.
+
+### Breaking strings apart
+
+In addition to building up strings out of components, strings may also be taken apart and broken into individual components. This is useful for tasks such as breaking apart a list of comma separated values into an array of the individual entities. Note that not all of these functions are in Swift's standard library and will require the importing of additional libraries. The most commonly used library is `Foundation`.
+
+```swift
+import Foundation
+
+let csv = "apple,pear,peach,orange,cherry,lime,goosberry"
+let fruit = csv.components(separatedBy: ",")
+// => ["apple", "pear", "peach", "orange", "cherry", "lime", "goosberry"]
```

Comment:
Is the `import` syntax introduced in this syntax? If so, it might be worth mentioning explicitly in the text that the `import` keyword is used to import functionality. This would also give this code example the unique bit of syntax for it to warrant inclusion in this document.